__tests__/model/model.query.test.js

```diff
@@ -4,7 +4,8 @@ const client = require('../../utils/apollo-test-client');
 const { newCategory, newModel } = require('./model.variables');
 require('dotenv').config();
 
-let modelId, categoryId, categoryName;
+let modelId, categoryId
+// , categoryName;
```

Comment:
Please, remove useless code

Reply:
Removed.
I know that this code was useless, but this component is not mine, so I commented it instead of completely remove it